src/test/java/core/basesyntax/service/CsvFileWriterTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.CsvFileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileWriterTest {
+    private static final FileWriter FILE_WRITER = new CsvFileWriter();
+
+    @Test
+    public void writeToFile_Ok() {
```

Comment:
Best practice of test naming is: methodName_stateUnderTest_expectedBehavior

Reply:
I agree here